.typos.toml

```diff
@@ -0,0 +1,11 @@
+[files]
```

Comment:
This file needs to be ignored in .gitattributes so that it's not present in the distribution.

Reply:
It is already ignored.

https://github.com/phpstan/phpstan/blob/7fab7a504376fc171e0787b0ca3d2fc2cf7026c6/.gitattributes#L1

Reply:
Oh right, thanks.